### PR TITLE
complete runtime error occur at int + string

### PR DESCRIPTION
**Pull request by garry911128**

### CODE REVIEW

File: mini-python-ocaml/compile.ml [hunk 1]
```diff
@@ -71,6 +82,38 @@ let arith_asm (code1:X86_64.text) (code2:X86_64.text) (instruction:X86_64.text)
...
+    call "exit" ++
+    (* No need for epilogue as the process will exit *)
+    nop
+  in
+  let data_code =
+    (* Data section for the error message *)
+    label "runtime_error_msg" ++
+    string error_message
+  in
+  text_code, data_code
```
Comment (kyynk) [inline mini-python-ocaml/compile.ml]:
```suggestion
let create_runtime_error () : X86_64.text * X86_64.data =
  let error_label = "runtime_error" in
  let error_message = "Runtime error occurred\n" in
  let text_code =
    label error_label ++
    (* Prologue *)
    pushq (reg rbp) ++
    movq (reg rsp) (reg rbp) ++
    (* Load the error message *)
    leaq (lab "runtime_error_msg") rdi ++
    (* Call printf to print the error message *)
    xorq (reg rax) (reg rax) ++
    call "printf" ++
    (* Exit with a non-zero status *)
    movq (imm 1) (reg rdi) ++
    call "exit" ++
    (* No need for epilogue as the process will exit *)
    nop
  in
  let data_code =
    (* Data section for the error message *)
    label "runtime_error_msg" ++
    string error_message
  in
  text_code, data_code
```

**Review by kyynk — CHANGES_REQUESTED** [review]
image

File: mini-python-ocaml/compile.ml (hunk 1 above)
Comment (garry911128) [reply mini-python-ocaml/compile.ml]: fixed

**Review by Vghxv — APPROVED** [review]
LGTM

**Review by kyynk — APPROVED** [review]
LGTM

**Pull request merged**